app/views/problems/_fields.html.erb

```diff
@@ -18,6 +18,6 @@ unchecked." %>
 <div class="action_buttons">
 	<%= f.submit "Save Problem" , {:class=>"btn primary"} %>
 	<% if @problem then %>
-	  <%= link_to "Delete this Problem", [@course, @assessment, @problem], method: :delete, class: 'btn btn-danger', data: { confirm: "Deleting will delete all associated problem data and cannot be undone. Are you sure you want to delete this problem?" } %>
+	  <%= link_to "Delete this Problem", [@course, @assessment, @problem], method: :delete, :style=> 'font-size:1em;', class: 'btn btn-danger', data: { confirm: "Deleting will delete all associated problem data and cannot be undone. Are you sure you want to delete this problem?" } %>
```

Comment:
The problem is that the Save Problem button is an `<input type="submit">` button while the Delete This Problem button is an `<a>` link formatted to look like a button using CSS, so they end up rendering differently even though they're both `btn` classes in CSS.

If it is possible to refactor this code a little so that both are `<a>` anchors that will be ideal because inline CSS is messy, but if that's not possible/too high effort let me know and I'll approve it

Reply:
Yup, I noted the tag difference in the context section. 

Changing the rendering is a bit difficult since the save button is utilizing forms for rendering while the delete button is simply utilizing the delete method. Thus, a better solution (instead of changing a bunch of code around) is to encapsulate the delete button `<a>` into a `<button>` tag in order to keep the same style as the `<input type="submit">`. That way, we can repeat this style for future buttons that are being rendered differently.